src/actions/make/make_csharp.lua

```diff
@@ -191,16 +192,17 @@
 		local makefile = path.getname(premake.filename(prj, ext))
 		local response = make.cs.getresponsefilename(prj)
 
+		local sep = os.is("windows") and "\\" or "/"
+
 		_p('$(RESPONSE): %s', makefile)
 		_p('\t@echo Generating response file', prj.name)
 
 		_p('ifeq (posix,$(SHELLTYPE))')
 			_x('\t$(SILENT) rm -f $(RESPONSE)')
 		_p('else')
-			_x('\t$(SILENT) if exist $(RESPONSE) del %s', path.translate(response, '\\'))
+			_x('\t$(SILENT) if exist $(RESPONSE) del %s', path.translate(response, sep))
```

Comment:
I think the premake code must ignore platform differences, so the <slash> and <backslash> should be hide, so modify this line code.